varcode/variant_collection.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2015. Mount Sinai School of Medicine
```

Comment:
Copyright (c) 2157. Mount Sinai School of Medicine 

(the Python3 vs Python4 schism will be mostly healed by then, with >20% Python4 adoption)